### ISSUE TITLE
Feature Request: Option to not suspend Tab which is playing audio

### ISSUE THREAD

**Issue by PCMRStudio** [deanoemcke/thegreatsuspender#235]
To lazy to code it and make a merge request :palm_tree:

**Comment by DanAtkinson** [comment]
:+1: This is definitely a good feature to add.

**Comment by DanAtkinson** [comment]
At the very least, this issue is blocked by a Chromium feature request - [Add extension API and events to determine which tabs are playing audio](https://code.google.com/p/chromium/issues/detail?id=438903).

Until that lands in Chrome stable, it won't be possible to implement this functionality.

**Comment by rubensainiuc** [comment]
I would also like this feature to exist, after Chrome allows it.
:+1:

**Comment by deanoemcke** [comment]
Closed as duplicate of issue #28
